control-plane/subcommand/server-acl-init/command_ent_test.go

```diff
@@ -386,15 +386,10 @@ func TestRun_ACLPolicyUpdates(t *testing.T) {
 				"anothertgw-policy",
 				"partitions-token",
 			}
+			time.Sleep(1 * time.Second)
 			policies, _, err = consul.ACL().PolicyList(nil)
 			require.NoError(t, err)
```

Comment:
~ `require.EventuallyWithT` instead of sleep?

Reply:
Ah, good catch. I was just playing around here, so I'll see if I can get away with removing it. There still might be an optimization here to wait for the bootstrap token, but it passes consistency on consecutive local runs.